en/pocket/privacy.ftl

```diff
@@ -0,0 +1,89 @@
+
+pocket-privacy-we-will = We will keep your information only for as long as we need it to provide you services, manage our business or as required by law or contract.
+pocket-privacy-were-a-global = We’re a global organization and our computers are in several different places around the world. We also use service providers whose computers may also be in various countries. This means that your information might end up on one of those computers in another country, and that country may have a different level of data protection regulation than yours. By giving us information, you consent to this kind of transfer of your information. No matter what country your information is in, we comply with applicable law and will also abide by the commitments we make in this privacy policy.
+pocket-privacy-we-have-agreements = We have agreements with third parties and have adopted other security measures in place to protect against the loss, misuse and/or unauthorized alteration of the information under our control or under the control of our service providers. Your personally identifiable information is protected by utilizing both online and offline security methods, including firewalls, passwords and restricted physical access to the places where your information is stored. Although we use industry standard practices to protect your privacy, we do not promise, and you should not expect, that your personal information or private communications will always remain free from security issues. In the event of a security breach involving your personal information, we will make any legally required disclosures to you in the most expedient time possible and without unreasonable delay, consistent with the legitimate interests of law enforcement, or any measures necessary to determine the scope of the breach and restore the reasonable integrity of the data system.
+pocket-privacy-identity-theft = Identity theft and the practice currently known as “phishing” are of great concern to { -brand-name-pocket }. Safeguarding information to help protect you from identity theft is a top priority. We do not and will not, at any time, request your username, password, credit card information or national identification number in an unsolicited e-mail or telephone communication. For more information about phishing, visit the Federal Trade Commission’s website.
```

Comment:
`e-mail` to `email`?

Reply:
This file has been moved to legal-docs. I'll make a separate PR to remove it from this repo

Reply:
The PR will have to come after this PR is merged, because it looks like the automation isn't handling files being deleted